OneCommunityGlobal/HighestGoodNetworkApp
# Tianyuan lint fix Service

# Description
- lint fix the Service folder 
- src/services/**


## Main changes explained:
- Remove Service Folder from .prettierignore and .eslintignore
- Lint checked all the files in the folder and manually fixed all of them
- Tested the project working normally

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. do `npx eslint src/services/** `
4. log as any user
5. go to /forgotpassword and check the forgetting password function worked normally (one of the service related to this component)
6. check the dashboard displaying correct time (one of the service related to user's timezone) by comparing it with what you see on our project dashboard(https://hgnapplication_react_beta.surge.sh/dashboard) 

## Screenshots or videos of changes:
<img width="1054" alt="PR_service" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73833864/c5216eff-7262-4c22-bace-7daa9ddb9e14">
<img width="1207" alt="Timeboard" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73833864/4e85e671-a91e-498a-b40b-eabe0545c29b">
<img width="394" alt="ForgetPasswordComponent" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73833864/c2f42f37-0c7f-438b-9783-fea94f05a6c2">
